code/modules/recycling/sortingmachinery.dm

```diff
@@ -243,3 +292,87 @@
 		var/n = text2num(href_list["nextTag"])
 		currTag = n
 	openwindow(usr)
+
+/obj/item/sales_tagger
+	name = "sales tagger"
+	desc = "A scanner that lets you tag wrapped items for sale, splitting the profit between you and cargo. Ctrl-Click to clear the registered account."

```

Comment:
Should "cargo" be capitalized or not? I forgot if we're supposed to capitalize department names.

Reply:
After close examination, no, cargo is not capitalized.